src/locales/de/translation.json

```
@@ -322,106 +322,106 @@
+            "Ich habe vor, kurz nach der Veröffentlichung separate Abschnitte für die Spielsteine und Miniaturen hinzuzufügen.",
+            "Aktuell sind die Kontrollkästchen nicht dauerhaft gespeichert. Wenn Sie die Seite aktualisieren oder verlassen, geht Ihr Fortschritt verloren. Entschuldigen Sie die Unannehmlichkeiten, während ich das kläre."
         ],
-        "legendHeader": "Legend",
+        "legendHeader": "Legende",
         "legendBody": {
-            "header": "If two columns: read left to right, top to bottom. This will ensure the running totals of cards spread across multiple packs (e.g. \"Monster cards: 11 (11/29)\") are read properly.",
+            "header": "Wenn zwei Spalten vorhanden sind: von links nach rechts, von oben nach unten lesen. Dadurch wird sichergestellt, dass die laufenden Summen von Karten, die sich auf mehrere Packs verteilen (z.B. 'Monsterkarten: 11 (11/29)'), ordnungsgemäß gelesen werden.",
             "body": [
                 {
                     "title": "TWOW(XX) Y/Z",
                     "details": [
-                        "This is the title of the gray card for each pack of cards. This is how I refer to each pack.",
-                        "XX is the arbitrary code that the supplier used to identify the groups of cards.",
-                        "Y is the current number of the pack out of total packs for code XX.",
-                        "Z is the total number of packs for value XX."
+                        "Dies ist der Titel der grauen Karte für jedes Kartenset. So beziehe ich mich auf jedes Set.",
+                        "XX ist der willkürliche Code, den der Lieferant zur Identifizierung der Kartenkartons verwendet hat.",
+                        "Y ist die aktuelle Nummer des Kartensets von insgesamt Kartensets für den Code XX.",
+                        "Z ist die Gesamtanzahl der Kartensets für den Wert XX."
                     ]
                 },
                 {
                     "title": "$t(common.expansions)",
                     "details": [
-                        "These are the expansions that the cards belong to in this pack."
+                        "Dies sind die Erweiterungen, zu denen die Karten in diesem Set gehören."
                     ]
                 },
                 {
-                    "title": "Example Type Cards: X (Y/Z) OR X/Z",
-                    "details": [
-                        "Example Type is the type of card, as referred to in Go On Board's \"Components\" lists.",
-                        "X is how many of the cards are found in this pack.",
-                        "Y is the current running total of this type of card across all packs. If excluded, all cards of this type were included in this pack",
-                        "Z is the total number of these cards you should have."
+                    "title": "Beispielkartentypen: X (Y/Z) ODER X/Z",
+                    "details": [                      
```

Nit: Clean up hanging whitespace, thank you!

```suggestion
                    "details": [
```